wpilibcExamples/src/main/cpp/commands/command2/ReplaceMeCommand2.h

```diff
@@ -17,6 +17,7 @@
 class ReplaceMeCommand2
     : public frc2::CommandHelper<frc2::Command, ReplaceMeCommand2> {
  public:
+  /* You should consider using the more terse Command factories API instead */
```

Comment:
I think a link to https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands would be helpful (also for C++)